CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+## 0.7.0 - 2024-04-24
+
+### Breaking Changes
+* **Drop `public` Schema Creation by Default**: In previous versions, the `materialize_database` resource automatically created a `public` schema in each new database, mimicking traditional SQL database behavior. Starting with version `v0.7.0`, this default behavior has been removed. Users must now explicitly define and manage `public` schemas within their Terraform configurations. This change is designed to align the Terraform provider's behavior more closely with its design principles, ensuring consistency and predictability.
```

Comment:
"Drop" is an overloaded and scary term here. While the underlying mechanism may have us dropping the schema immediately after creating the database, we are not going to do this to existing database resources. Maybe something like

> `public` schemas are no longer created by default: In previous versions...

?